### PR TITLE
@CeramicStudio/web-playground#32 - Update Web Playground UI

### PR DESCRIPTION
**Pull request by Sterahi**
Added the following components: 
- Visual indicator for DID
- Loading Indicator for `Connect Wallet` button.
- Add Success & Error alerts for 3ID-connect messages.
- Hide QuickStart instructions until user has authenticated

General improvements:
Small UI tweaks to make space more readable. 

Sample: 
**User Not Authenticated:**
Screen Shot 2021-07-23 at 1 53 01 PM

**User Authenticated:**
Screen Shot 2021-07-23 at 1 52 52 PM

**All new elements Enabled:**
Screen Shot 2021-07-23 at 1 55 37 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,70 +13,110 @@
...
   </head>
 
   <body>
-    <div id="root"></div>
-    <header>
-      <div>
-        <img id="logo" src="images/web-playground-logo.svg" alt="">
+    <div class = "container">
+      <header>
+        <div class = 'row'>
```
Comment (PaulLeCam) [inline index.html:79]: The formatting seems super inconsistent between the use of single or double quotes and spacing. Maybe you could could configure Prettier in your IDE so it formats the code automatically?

File: quickstart.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  - Notice that we did not provide our DID, but it's updating our record. This is because we default to the currently active ID.
+  - These fields are optional, but they can be set to be required by the schema.
+3. `await idx.merge('basicProfile', { emoji: '👍🏻'})`
+   - Updating records is easy, change any details you need to anytime you need to.
+
+## Creating a Custom Schema:
+Schema creation might seem daunting at first, but it's quite simple when you understand how it's all built. 
+
+### Helper Functions:
+To start we will need to create a few helper functions. These are all included in our `@ceramicstudio/idx-tools` package, this is just to demystify them. To use them you can just copy & paste the code into your browser console.
```
Comment (PaulLeCam) [inline quickstart.md:25]: These are internal functions in IDX tools, I'd rather we avoid using/referencing them elsewhere please as they are not documented and could lead to more confusion.
The tile creation can be simply `const doc = await TileDocument.create(ceramic, content, metadata)`, the rest are sanity checks.

File: quickstart.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      /**
+      	* Definitions are a key part of how Ceramic categorizes data.
+      	* Without the definition the schema is inaccessible. 
+      	*/
+      const createDefinition = async (ceramic, definition) => {
+      	// We use our createTile helper to create our definition tile. 
+      	return await createTile(ceramic, 
+      		definition, 
+      		{
+      			schema: 'ceramic://k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw'
```
Comment (PaulLeCam) [inline quickstart.md:73]: Might be safer to import the definition schema URL from `idx-constants` here, no?

File: quickstart.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      		})
+      }
+    ```
+
+### Creating & Using Custom Schemas
+Simply put a schema is just a JSON object. The following will create a super simple blog post schema. 
+```JSON
```
Comment (PaulLeCam) [inline quickstart.md:80]: Should be JS here, not JSON

File: quickstart.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        maxLength: 4000,
+      },
+    },
+  }
+```
+You can create any schema you want, but we'll be using the above schema for the guide.
+
+1. Create Schema: `const schema = await createSchema(ceramic, { controllers: [ceramic.did.id], content: blogSchema})`
+   - Note that we're using our createSchema function from the Helper Functions section.
+   - The controllers parameter is optional and will default to the current DID. However, if you need more than one DID to control the schema (and any content created) they need to be listed in the controllers section.
```
Comment (PaulLeCam) [inline quickstart.md:109]: That seems misleading as we only support having one controller currently.

**Review by PaulLeCam — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sterahi) [reply index.html:79]: Good idea. Will implement it.

File: quickstart.md (hunk 2 above)
Comment (Sterahi) [reply quickstart.md:25]: This file shouldn't be here. Totally my mistake for including it. This is more for the "quickstart" documentation. I need to do some testing to make sure that things behave properly with the IDX tools and whatnot.

**Review by PaulLeCam — APPROVED** [review]
Thanks for the changes!

**Pull request merged**